### PR TITLE
Updated docstring in deleteMessage methods

### LINKED ISSUE #617: May 18 changes
### Payment:
- [x] New updates: shipping_query and pre_checkout_query
- [x] New message content fields: invoice and successful_payment
- [x] New bot methods: sendInvoice, answerShippingQuery, and answerPreCheckoutQuery
- [x] New button type: pay (new boolean argument for InlineKeyBoardButton)

### Weird rounded video messages
- [x] New bot method: sendVideoNote
- [x] New Message field: video_note
- [x] New ChatActions: record_video_note or upload_video_note
- [x] VideoNote class

### i18n
- [x] New User field: language_code
Note that the documentation uses "may" wording here... we should do some research as to when.

### Admin power???
- [x] Remove warning from delete_message

### Minor stuff
- [x] Rename new_chat_member to new_chat_member**s**
- [x] New fields gif_duration (InlineQueryResultGif) and mpeg4_duration (InlineQueryResultMpeg4Gif)

### PR DESCRIPTION
**Pull request by evgfilim1**
Fixes #617 a bit

### CODE REVIEW

**Comment by bomjacob** [comment]
Could you update the docstring to include the description of the bot api while you're at it?

**Comment by evgfilim1** [comment]
@bomjacob I should copy method description from Bot API documentation and put it in docstring, right?

**Comment by jh0ker** [comment]
@evgfilim1 Yes

**Comment by evgfilim1** [comment]
@bomjacob @jh0ker Done

File: telegram/bot.py [hunk 1]
```diff
@@ -1159,6 +1161,7 @@ def edit_message_text(self,
             message_id (Optional[int]): Required if inline_message_id is not specified. Unique
                 identifier of the sent message.
             inline_message_id (Optional[str]): Required if chat_id and message_id are not
+            inline_message_id (Optional[str]): Required if chat_id and message_id are not
```
Comment (jh0ker) [inline telegram/bot.py]: I think you accidentally a line

**Comment by evgfilim1** [comment]
@jh0ker Fixed

**Pull request merged**